apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| -------------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| cdn_asset_uris             | Table  | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                             |
+| collection_id              | String | Unique identifier for the collection. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b88"                         |
+| collection_name            | String | The formal name of the collection. Ex. "Mythic Dragons"                                                                                |
+| creator_address            | String | This is an Aptos account address that created the collection. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| current_supply             | bigint | Current supply of tokens in this collection. Ex. 500                                                                                   |
+| description                | String | Description of the collection. Ex. "A collection of rare digital dragons."                                                             |
+| last_transaction_timestamp | String | Timestamp of the last transaction involving this collection. Ex. "2024-04-17T02:14:25.68771"                                           |
+| last_transaction_version   | bigint | Blockchain version of the last transaction involving this collection. Ex. 3000000002                                                   |
+| max_supply                 | bigint | Maximum possible quantity of tokens in this collection. Ex. 1000                                                                       |
+| mutable_description        | String | Changeable description of the collection. Ex. "Updated collection description."                                                        |
+| mutable_uri                | String | URI where updated information about the collection can be found. Ex. "https://example.com/collections/updated-info"                    |
+| table_handle_v1            | String | Legacy identifier handle for the collection in earlier schema versions. Ex. "handle_12345"                                             |
+| token_standard             | String | Aptos standard that the collection adheres to. Ex. "v1"                                                                                |
+| total_minted_v2            | bigint | Total number of tokens minted in this collection under schema version 2. Ex. 800                                                       |
+| uri                        | String | URI linking to further information about the collection. Ex. "https://example.com/collections/9876543210"                              |
```

Comment:
Helpful to point out that this is where the image lives and can be a json as well.